server/server.go

```diff
@@ -110,6 +111,11 @@ func Start(ctx context.Context, logger *zap.Logger, cfg *Config) error {
 	}
 
 	app := http.HandlerFunc(proxy.handler)
+	if cfg.AuthVerify {
+		http.Handle(cfg.AuthVerifyPath, authVerify(middleware))
+	}
+
+	http.Handle("/saml/sign_in", http.HandlerFunc(middleware.HandleStartAuthFlow))
```

Comment:
Can you point me to docs (or oauth2-proxy source code) that shows how this flow proceeds?

Reply:
https://oauth2-proxy.github.io/oauth2-proxy/features/endpoints/

Traefik specific example:
https://github.com/oauth2-proxy/oauth2-proxy/blob/master/contrib/local-environment/traefik/dynamic.yaml